tests/phpunit/integration/Core/Conversion_Tracking/Conversion_Event_Providers/WooCommerceTest.php

```diff
@@ -30,14 +27,15 @@ public function set_up() {
 		$this->woocommerce = new WooCommerce( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
 	}
 
-	public function tear_down() {
-		parent::tear_down();
-		$this->deactivate_all_test_plugins();
-	}
-
+	/**
+	 * @runInSeparateProcess
+	 */
 	public function test_is_active() {
 		$this->assertFalse( $this->woocommerce->is_active() );
-		$this->activate_plugin( 'woocommerce/woocommerce.php' );
```

Comment:
In the future if we move away from runtime active checks, this would be the way to go.